sdk/eventgrid/Azure.Messaging.EventGrid/src/Customization/EventGridModelFactory.cs

```diff
@@ -437,7 +437,25 @@ public static MediaLiveEventChannelArchiveHeartbeatEventData MediaLiveEventChann
         [EditorBrowsable(EditorBrowsableState.Never)]
         public static AcsRecordingFileStatusUpdatedEventData AcsRecordingFileStatusUpdatedEventData(AcsRecordingStorageInfoProperties recordingStorageInfo, DateTimeOffset? recordingStartTime, long? recordingDurationMs, string sessionEndReason)
         {
-            return AcsRecordingFileStatusUpdatedEventData(recordingStorageInfo: recordingStorageInfo, recordingStartTime: recordingStartTime, recordingDurationMs: recordingDurationMs, sessionEndReason: sessionEndReason, recordingChannelType: null);
+            return AcsRecordingFileStatusUpdatedEventData(recordingStorageInfo: recordingStorageInfo, recordingStartTime: recordingStartTime, recordingDurationMs: recordingDurationMs, sessionEndReason: sessionEndReason, channelType: null);
+        }
+
+        /// <summary> Initializes a new instance of AcsRecordingFileStatusUpdatedEventData. </summary>
+        /// <param name="recordingStorageInfo"> The details of recording storage information. </param>
+        /// <param name="recordingStartTime"> The time at which the recording started. </param>
+        /// <param name="recordingDurationMs"> The recording duration in milliseconds. </param>
+        /// <param name="recordingContentType"> The recording content type- AudioVideo, or Audio. </param>
+        /// <param name="recordingChannelType"> The recording  channel type - Mixed, Unmixed. </param>
+        /// <param name="recordingFormatType"> The recording format type - Mp4, Mp3, Wav. </param>
```

Comment:
```suggestion
        /// <param name="recordingFormatType"> The recording format type.  For example, Mp4, Mp3, Wav. </param>
```

We may want to be explicit about these being examples rather than an always-up-to-date exhaustive list.